blank-canvas/functions.php

```diff
@@ -71,25 +71,55 @@ function blank_canvas_setup() {
  */
 function blank_canvas_remove_parent_theme_features() {
 
-	// Theme Support
+	// Theme Support.
 	remove_theme_support( 'custom-header' );
-	remove_theme_support( 'custom-logo' );
 	remove_theme_support( 'customize-selective-refresh-widgets' );
 
-	// Navigation Areas
+	// Navigation Areas.
 	unregister_nav_menu( 'primary' );
 	unregister_nav_menu( 'footer' );
 	unregister_nav_menu( 'social' );
 }
-add_action( 'after_setup_theme', 'blank_canvas_remove_parent_theme_features', 10 );
+add_action( 'after_setup_theme', 'blank_canvas_remove_parent_theme_features', 11 );
 
+/**
+ * Dequeue Seedlet scripts.
+ */
 function blank_canvas_dequeue_parent_scripts() {
-	// Naviation assets
+
+	// Naviation assets.
 	wp_dequeue_script( 'seedlet-primary-navigation-script' );
 	wp_dequeue_style( 'seedlet-style-navigation' );
 }
 add_action( 'wp_enqueue_scripts', 'blank_canvas_dequeue_parent_scripts', 11 );
 
+/**
+ * Remove Seedlet's widget area.
+ */
+function blank_canvas_remove_widgets_area() {
+	unregister_sidebar( 'sidebar-1' );
+}
+add_action( 'widgets_init', 'blank_canvas_remove_widgets_area', 11 );
+
+/**
+ * Remove unused custmizer settings.
+ */
+function blank_canvas_remove_customizer_settings( $wp_customize ) {
+
+	// Remove the navigation menus Customizer panel.
+	$wp_customize->get_panel( 'nav_menus' )->active_callback = '__return_false';
```

Comment:
I expected to be able to use something like `$wp_customize->remove_panel( 'nav_menus' )`, but it throws an error:

> Notice: WP_Customize_Manager::remove_panel was called incorrectly. Removing nav_menus manually will cause PHP warnings. Use the customize_loaded_components filter instead. Please see Debugging in WordPress for more information. (This message was added in version 4.5.0.)

The approach I used instead [comes from this thread](https://wordpress.stackexchange.com/a/225234), and seems to work in my testing.